test/basic.test.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+const { test } = require('tap')
+const rimraf = require('rimraf')
+const ClinicFlame = require('../index.js')
+
+test('cmd - test collect - data exists', function (t) {
+  const tool = new ClinicFlame()
+
+  function cleanup (err, dirname) {
+    t.ifError(err)
+
+    let count = 0
+    function callback (err) {
+      t.ifError(err)
+      if (++count === 2) {
+        t.end()
+      }
+    }
+    rimraf(dirname, callback)
```

Comment:
i think you might wanna add an assert in from of this :D just so we don't accidentally rm -rf someones disk when we have a failing test

Reply:
This should not happen because of the `t.ifError(err)` call. If there's no dirname, there should be an error. In that case the function will fail early on.

Reply:
I'd still be a lot more confortable if this asserted that __dirname is inside the test dir, in case someone messes up the dirname argument inside flame.

Reply:
Done. This pattern has been used a lot everywhere throughout clinic so far and that's why I used that as well.